platforms/core-configuration/file-collections/src/integTest/groovy/org/gradle/api/file/FileTreeRelativePathIntegrationTest.groovy

```diff
@@ -0,0 +1,60 @@
+/*
+ * Copyright 2025 the original author or authors.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package org.gradle.api.file
+
+import org.gradle.integtests.fixtures.AbstractIntegrationSpec
+import org.gradle.test.precondition.Requires
+import org.gradle.test.preconditions.IntegTestPreconditions
+import spock.lang.Issue
+
+class FileTreeRelativePathIntegrationTest extends AbstractIntegrationSpec {
+
+    // Tests a use case that will be forbidden by IP
+    @Requires(IntegTestPreconditions.NotIsolatedProjects)
+    @Issue("https://github.com/gradle/gradle/issues/30052")
+    def "ConfigurableFileTree supports setting directory to relative path at execution time"() {
```

Comment:
🤔 This test seems like a duplication of `ConfigurableFileCollection`'s test with same scenario. Doesn't it make sense to merge them in a data-driven test?

Reply:
They are very similar, though different in a number of subtle ways. I presume that's the reason why most other integration tests in the package are split into different files for CFC and CFT.

I don't feel strongly. After trying out your suggestion, it does not look too bad. We can have a test class dedicated to relative file paths then.

I'll push the changes, following your suggestion